### PR TITLE
feat: 26 & 34 home page enhance

### PR DESCRIPTION
**Pull request by yunshen513**
### Descriptions
enhance home page

- Link to UI design (if possible):

### Provide screenshots or videos that the function is working.
**Desktop**
image

**Tablet**
image

**Phone**
image

### Provide screenshots of no warning, or error in the console.
image

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,6 @@
     "@types/react-dom": "18.0.11",
     "eslint": "8.35.0",
     "eslint-config-next": "13.2.1",
-    "msw": "^1.1.0",
```
Comment (tatoMa) [inline package.json:33]: Again, don't do anything unrelated to the ticket. I remember we have discussed this and we both agree to remove this msw lib, just a little reminder here.

**Review by tatoMa — COMMENTED** [review]
The size of each album is different from UI design I believe
image

File: package.json (hunk 1 above)
Comment (tatoMa) [reply package.json:33]: The album size will always same, the different is the gap between them if the responsive situation happens.

**Review by tatoMa — COMMENTED** [review]
Consider trim the text as you do on the albumItem.
image

File: src/pages/index.tsx [hunk 2]
```diff
@@ -22,28 +16,21 @@ export default function Home({ newAlbums }: HomeProps) {
...
-  //   type: "newSongs",
-  //   songs: newSongsRes.status ? newSongsRes.songs : [],
-  // };
-
-  const newAlbumsEA = await getNewAlbums("EA", 2);
-  const newAlbumsJP = await getNewAlbums("JP", 2);
-  const newAlbumsKR = await getNewAlbums("KR", 1);
-  const newAlbumsZH = await getNewAlbums("ZH", 2);
-  const newAlbums = [...newAlbumsEA, ...newAlbumsJP, ...newAlbumsKR, ...newAlbumsZH];
+  const areas = ["EA", "JP", "KR", "ZH"];
```
Comment (tatoMa) [inline src/pages/index.tsx:27]: This areas variable can be inside of a const file.

File: src/pages/index.tsx [hunk 3]
```diff
@@ -22,28 +16,21 @@ export default function Home({ newAlbums }: HomeProps) {
...
+  const newAlbums = await Promise.all(
```
Comment (tatoMa) [inline src/pages/index.tsx:28]: At least there would be an error handler or use promise.allSettled.

File: src/pages/index.tsx [hunk 4]
```diff
@@ -1,16 +1,10 @@
...
 import Head from "next/head";
-import getNewSongs from "../utils/getNewSongs";
 import { MusicList } from "types";
 import getNewAlbums from "@/utils/getNewAlbums";
 import Albums from "@/components/Albums";
 
 type HomeProps = {
-  // newSongsList: MusicList;
-  newAlbums: MusicList[];
+  newAlbums: { area: "EA" | "JP" | "KR" | "ZH"; albums: MusicList[] }[];
```
Comment (tatoMa) [inline src/pages/index.tsx:7]: Same, areas inside of a const file.

File: src/layouts/index.tsx [hunk 5]
```diff
@@ -1,21 +1,22 @@
...
-      <div className="flex h-full w-full flex-col md:w-[calc(100vw-64px)] lg:w-[calc(100vw-320px)]">
-        <SearchBar />
-        <main className="mb-[150px] h-[calc(100vh-150px)] w-full bg-dark px-2 md:mb-[120px] md:h-[calc(100vh-120px)] md:px-10">
+      <div className="flex h-screen w-4/5 flex-col bg-dark">
+        <div className="grow overflow-y-scroll px-2 [-ms-overflow-style:'none'] [scrollbar-width:'none'] md:px-10 [&::-webkit-scrollbar]:hidden">
+          <SearchBar />
+          <div className="my-8 h-1/3">This is banner</div>
           {children}
-        </main>
+        </div>
```
Comment (tatoMa) [inline src/layouts/index.tsx:19]: Why removed main?

**Comment by yunshen513** [comment]
Sorry, I thought no one would continue doing this project, so I changed as much as I noticed to improve it and caused a lot unrelated changes. I will close this request to keep the code clean.

**Pull request closed without merging**